wandb/sdk/launch/deploys/aws-eks/launch-agent.yaml

```diff
@@ -16,7 +16,7 @@ spec:
       serviceAccountName: wandb-launch-serviceaccount
       containers:
       - name: launch-agent
-        image: wandb/launch-agent-dev:latest
+        image: wandb/launch-agent-dev:939c8c754b
```

Comment:
also why?

Reply:
I just want our deployment code to reflect the latest tested release of the agent at any given time.

Reply:
wish there were a clean way to set these up as configurable constants somewhere...